autonat/autonat-v2.md

```diff
@@ -19,71 +19,89 @@ Interest Group: [@marten-seemann], [@marcopolo], [@mxinden]
 
 A priori, a node cannot know if it is behind a NAT / firewall or if it is
 publicly reachable. Knowing its NAT status is essential for the node to be
-well-behaved in the network: A node that's behind a doesn't need to
-advertise its (undialable) addresses to the rest of the network, preventing
+well-behaved in the network: A node that's behind a NAT / firewall doesn't need
+to advertise its (undialable) addresses to the rest of the network, preventing
 superfluous dials from other peers. Furthermore, it might actively seek to
 improve its connectivity by finding a relay server, which would allow other
 peers to establish a relayed connection.
 
-`autonat v2` allows nodes to determine reachability for individual addresses. 
-Using `autonat v2` nodes can build an address pipeline where they can test 
-individual addresses discovered by different sources like identify, upnp 
-mappings, circuit addresses etc for reachability.
+In `autonat v2` client sends a priority ordered list of addresses. On receiving
+this list the server dials the first address on the list that it is capable of
+dialing. `autonat v2` allows nodes to determine reachability for individual
+addresses. Using `autonat v2` nodes can build an address pipeline where they can
+test individual addresses discovered by different sources like identify, upnp
+mappings, circuit addresses etc for reachability. Having a priority ordered list
+of addresses provides the ability to verify low priority addresses.
```

Comment:
Should this mention that this helps with the rollout of new transports that are only supported by few nodes in the network?

Reply:
I added text that explains this case.